packages/components/src/components/hds/form/section/multi-field-group/index.hbs

```diff
@@ -0,0 +1,8 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: MPL-2.0
+}}
+
+<Hds::Layout::Flex class="hds-form__section-multi-field-group" @tag="div" @gap="16" @align="end" ...attributes>{{yield
```

Comment:
[todo] I would not mix&match `Hds::Layout::Flex` with a custom `Item`, I would probably just convert this to a custom flex (we already have the class name, and it's just 3 properties to declare)

Reply:
Ok, well the custom Item uses the `Layout::Flex::Item` though. So I'd have to make both a fully custom Flex parent and a fully custom Item...

Reply:
I took out/replaced the Flex::Layout components